core/src/main/java/io/questdb/cairo/wal/WalEventWriter.java

```diff
@@ -196,6 +197,9 @@ private void appendFunctionValue(Function function) {
             case ColumnType.UUID:
                 eventMem.putLong128(function.getLong128Lo(null), function.getLong128Hi(null));
                 break;
+            case ColumnType.ARRAY:
```

Comment:
oops - update SQL array bind variable will fail here

Reply:
Fixed in https://github.com/questdb/questdb/pull/5243/commits/67c24541906916ef81200058ffa2d58403058518